### PR TITLE
Update the README.md file to describe the repo.

### PR DESCRIPTION
**Pull request by aliu917**
I updated the readme as the prompts instructed.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
```
Comment (dibenede) [inline portfolio/src/main/webapp/script.js]: Rename greeting -> fact here and below

**Review by dibenede — CHANGES_REQUESTED** [review]
I recommend keeping the commit message as descriptive as possible. Git typically uses the first 80 characters before a newline as the brief message that will show up in the short commit log. You can then expand on it for the extended history view. So, for this change, I would do something like:

Add and format data about me (I think that's less than 80)

* Add a summary about me.
* Make information about my hobbies available on click.
* Add basic formatting.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -10,14 +10,27 @@
     <div id="content">
       <h1>Angela's Portfolio</h1>
       <a href="https://github.com/aliu917">
-      <img border="0" alt="Github" src="images/github.png" width="50" height="50">
+        <img border="0" alt="Github" src="images/github.png" width="50" height="50">
       </a>
-      <p></p>
```
Comment (dibenede) [inline portfolio/src/main/webapp/index.html]: optional: I think you can just do ```<p />``` for things like this

**Review by dibenede — APPROVED** [review]

**Review by dibenede — APPROVED** [review]

**Comment by aliu917** [comment]
A question about the commit messages: are you saying that each commit should contain the change I made for that commit in the first line and then bullet points for all the previous commit messages I made? So like for my most recent commit about adding a project section, it would be:
Finished the project section

- changed design and formatting
- add a summary about me
- add basic formatting
- make information about hobbies available on click

**Comment by dibenede** [comment]
> A question about the commit messages: are you saying that each commit should contain the change I made for that commit in the first line and then bullet points for all the previous commit messages I made? So like for my most raecent commit about adding a project section, it would be:
> Finished the project section
> 
> * changed design and formatting
> * add a summary about me
> * add basic formatting
> * make information about hobbies available on click

No, sorry. I may have put my comment in a weird spot. Basically your first line should be a summary. If you have a small commit, the summary may be sufficient to describe what you did and you can call it good.

If you have a large commit, then it's a good idea to list out what's included in it. Sorry for the confusion.

**Pull request merged**